lib/statsd.js

```diff
@@ -64,7 +65,7 @@ const Client = function (host, port, prefix, suffix, globalize, cacheDns, mock,
   this.telegraf = options.telegraf || false;
   this.maxBufferSize = options.maxBufferSize || 0;
   this.sampleRate = options.sampleRate || 1;
-  this.bufferFlushInterval = options.bufferFlushInterval || 1000;
+  this.bufferFlushInterval = options.bufferFlushInterval || (process.env.NODE_ENV === 'test' ? 10 : 1000);
```

Comment:
This makes tests to run a lot faster.

Reply:
Then I think the tests should set it explicitly -- I hate it (as application developer) when underlying libraries changes behaviour between environments...

Reply:
As mentioned - now it's set explicitly in testing/mocking code.